src/util/Util.js

```diff
@@ -56,7 +56,7 @@ class Util {
    * @returns {string|string[]}
```

Comment:
As this now only return an array:
```suggestion
    * @returns {string[]}
```